### ISSUE TITLE
Loading fails everytime

### ISSUE THREAD

**Issue by Bennyboy1695** [hilburn/QuestSheets#1]
Upon using the load command it fails and brings up a nice stacktrace in your log files.
https://gist.github.com/Bennyboy1695/4eeb714268f7e0a25ed3

**Comment by ChetwyndKid** [comment]
Occasionally it will crash when opening the questbook after this as well.

**Comment by hilburn** [comment]
Are you A) loading a set of quests that's already in the book, or B) loading a set of quests you have modified yourself?

**Comment by ChetwyndKid** [comment]
Loading Modified quests

**Comment by lorddusk** [comment]
It;s because it will also try and load the config files and .hqm file itself.

(This is only the case when the json files are in the HQM config folder)

**Comment by ChetwyndKid** [comment]
So if I remove the .hqm file it should work?

**Comment by hilburn** [comment]
It will not load any quests in a group that already exists in the book

**Comment by hilburn** [comment]
Of course, you may have just broken it by trying to modify it but inputting invalid data

**Comment by ChetwyndKid** [comment]
Alright I shall have a look at this in a sec.

**Comment by ChetwyndKid** [comment]
Works now, Thank you.
